File: app/api/token/token.py
# pip3 install requests
import requests
import json

API_KEY_SECRET = "bytepixelsfu_default_secret"
bytepixel_URL = "https://sfu.bytepixel.com/api/v1/token"
#bytepixel_URL = "http://localhost:3010/api/v1/token"

headers = {
    "authorization": API_KEY_SECRET,
    "Content-Type": "application/json",
}

data = {
    "username": "username",
    "password": "password",
    "presenter": "true",
    "expire": "1h"
}

response = requests.post(
    bytepixel_URL, 
    headers=headers, 
    json=data
)

print("Status code:", response.status_code)
data = json.loads(response.text)
print("token:", data["token"])

File: app/api/token/token.php
<?php

$API_KEY_SECRET = "bytepixelsfu_default_secret";
$bytepixel_URL = "https://sfu.bytepixel.com/api/v1/token";
#$bytepixel_URL = "http://localhost:3010/api/v1/token";

$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, $bytepixel_URL);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
curl_setopt($ch, CURLOPT_POST, 1);

$headers = [
    'authorization:' . $API_KEY_SECRET,
    'Content-Type: application/json'
];

curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);

$data = array(
    "username"      => "username",
    "password"      => "password",
    "presenter"     => true,
    "expire"        => "1h",
);

$data_string = json_encode($data);

curl_setopt($ch, CURLOPT_POSTFIELDS, $data_string);

$response = curl_exec($ch);
$httpcode = curl_getinfo($ch, CURLINFO_HTTP_CODE);

curl_close($ch);

echo "Status code: $httpcode \n";
$data = json_decode($response);
echo "token: ", $data->{'token'}, "\n";
